src/main/mw.c

```diff
@@ -729,8 +746,8 @@ void loop(void)
     }
 
     currentTime = micros();
-    if (masterConfig.looptime == 0 || (int32_t)(currentTime - loopTime) >= 0) {
```

Comment:
`masterConfig.looptime == 0` code path was hidden in looptime calculation ... but it's probably OK